diff --git a/node_modules/@firebase/auth/dist/rn/index.js b/node_modules/@firebase/auth/dist/rn/index.js
index 5766fd1..184d6e8 100644
--- a/node_modules/@firebase/auth/dist/rn/index.js
+++ b/node_modules/@firebase/auth/dist/rn/index.js
@@ -3,6 +3,7 @@
 Object.defineProperty(exports, '__esModule', { value: true });
 
 var reactNative = require('react-native');
+var AsyncStorage = require('../../../../@react-native-async-storage/async-storage');
 var app = require('@firebase/app');
 var phone = require('./phone-1d904137.js');
 var tslib = require('tslib');
@@ -124,7 +125,7 @@ function getReactNativePersistence(storage) {
  *
  * @public
  */
-var reactNativeLocalPersistence = getReactNativePersistence(reactNative.AsyncStorage);
+var reactNativeLocalPersistence = getReactNativePersistence(AsyncStorage);
 function getAuth(app$1) {
     if (app$1 === void 0) { app$1 = app.getApp(); }
     var provider = app._getProvider(app$1, 'auth');
